test/plugin/base_test.rb

```diff
@@ -2292,9 +2404,22 @@ def verify_subfields_when_not_hash(payload_key)
       d.run
     end
     verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
-      field = get_fields(entry['jsonPayload'])[payload_key]
-      assert_equal 'a_string', get_string(field), entry
-      assert_false entry.key?(destination_key), entry
+      if payload_key == DEFAULT_LABELS_KEY
+        # The malformed field has been removed from the payload.
+        assert_true get_fields(entry['jsonPayload']).empty?, entry
+        # No additional labels.
+        assert_equal COMPUTE_PARAMS[:labels].size,
+                     entry[destination_key].size, entry
+      else
+        # Leave the malformed field as it is.
+        # TODO(qingling128) In the next breaking change release, make the older
```

Comment:
breaking change -> "on the next major after x.y.z", so when someone looks at this in the future the can more easily make a decision whether it's OK to make this change.

Reply:
Makes sense. Changed.